### PR TITLE
first PR

### PR DESCRIPTION
**Pull request by RGB-Seif**

### CODE REVIEW

File: src/main/java/RegisterPage.java [hunk 1]
```diff
@@ -0,0 +1,273 @@
...
+        driver.element().click(newsLetterCheckBox);
+        return this ;
+    }
+
+    public RegisterPage clickOnReceiveOffersCheckBox() {
+        driver.element().click(offersCheckBox);
+        return this ;
+    }
+
+    public RegisterPage fillFirstName(String firstN) {
```
Comment (Mohamed-M-Selim) [inline src/main/java/RegisterPage.java]: firstN ?? what is the meaning of it ?

File: src/main/java/RegisterPage.java [hunk 2]
```diff
@@ -0,0 +1,273 @@
...
+        driver.element().type(firstNameField, firstN);
+        return this ;
+    }
+
+    public RegisterPage fillLastName(String lastN) {
```
Comment (Mohamed-M-Selim) [inline src/main/java/RegisterPage.java]: lastN ?? what is the meaning of it ?

File: src/main/java/RegisterPage.java [hunk 3]
```diff
@@ -0,0 +1,273 @@
...
+    public RegisterPage fillAddress(String addressOne) {
+        driver.element().type(addressOneField,addressOne);
+        return this ;
+    }
+
+    public RegisterPage fillAddressTwo(String addressTwo) {
+        driver.element().type(adressTwoField,addressTwo);
+        return this ;
+    }
+    public RegisterPage fillAddressData(String firstN, String lastN,
```
Comment (Mohamed-M-Selim) [inline src/main/java/RegisterPage.java]: firstN , lastN ?? what is the meaning of them ?

File: src/main/java/RegisterPage.java (hunk 1 above)
Comment (RGB-Seif) [reply src/main/java/RegisterPage.java]: this comment was resolved

File: src/main/java/RegisterPage.java (hunk 2 above)
Comment (RGB-Seif) [reply src/main/java/RegisterPage.java]: this comment was resolved

File: src/main/java/RegisterPage.java (hunk 3 above)
Comment (RGB-Seif) [reply src/main/java/RegisterPage.java]: this comment was resolved

**Review by Mohamed-M-Selim — APPROVED** [review]

**Pull request merged**